bundle/manifests/dell-csm-operator.clusterserviceversion.yaml

```diff
@@ -14,7 +14,7 @@ metadata:
           "spec": {
             "client": {
               "common": {
-                "image": "dellemc/connectivity-client-docker-k8s:1.19.0",
+                "image": "docker.io/dellemc/connectivity-client-docker-k8s:1.19.0",
```

Comment:
These are ANK8S components and should be removed

Reply:
Hi @hoppea2 As per the info we got sometime back, it is mentioned that they are going to stay this release.. I don't have any idea if that changed later.